Packs/Okta/Integrations/Okta_IAM/Okta_IAM_test.py

```diff
@@ -0,0 +1,129 @@
+import demistomock as demisto
+from requests import Response, Session
+from Okta_IAM import Client, get_user_command  # , create_user_command, update_user_command, enable_user_command
+from CommonServerPython import IAMErrors, IAMUserProfile
+
+''' ARGUMENTS '''
+
+USER_ARGS = {
+    'user-profile': {
+        'email': 'testdemisto2@paloaltonetworks.com'
+    }
+}
+
+
+''' OUTPUTS '''
+
+GET_USER_OUTPUT__EXISTING_USER = {
+    "id": "mock_id",
+    "status": "PROVISIONED",
+    "profile": {
+        "firstName": "mock_first_name",
+        "lastName": "mock_last_name",
+        "login": "testdemisto2@paloaltonetworks.com",
+        "email": "testdemisto2@paloaltonetworks.com"
+    }
+}
+
+GET_USER_REQUEST__BAD_RESPONSE = Response()
+GET_USER_REQUEST__BAD_RESPONSE.status_code = 500
+GET_USER_REQUEST__BAD_RESPONSE._content = b'{"errorCode": "mock_error_code", ' \
+                                          b'"errorSummary": "mock_error_summary", ' \
+                                          b'"errorCauses": [{"errorSummary": "reason_1"}, ' \
+                                          b'{"errorSummary": "reason_2"}]}'
+
+
+def mock_client():
+    client = Client(
+        base_url='https://test.com',
+        verify=False,
+        token='test',
+        proxy=False
+    )
+    return client
+
+
+def get_outputs_from_user_profile(user_profile):
+    entry_context = user_profile.to_entry()
+    outputs = entry_context.get('Contents')
+    return outputs
+
+
+def test_get_user_command__existing_user(mocker):
+    """
+    Given:
+        - An Okta IAM client object
+        - A user-profile argument that contains an email of an existing user in Okta
+    When:
+        - Calling function get_user_command
+    Then:
+        - Ensure the resulted User Profile object holds the correct user details
+    """
+    client = mock_client()
+    args = USER_ARGS
```

Comment:
don't use constants in these cases. The test arguments should be clear to the reader